source/globalCommands.py

```diff
@@ -3593,6 +3593,22 @@ def script_braille_toggleShowCursor(self, gesture):
 		braille.handler._updateDisplay()
 		ui.message(state)
 
+	@script(
+		# Translators: Input help mode message for speak on routing command.
+		description=_("Toggles speaking the character under the cursor when routing cursor in text"),
+		category=SCRCAT_BRAILLE,
+	)
+	@gui.blockAction.when(gui.blockAction.Context.BRAILLE_MODE_SPEECH_OUTPUT)
+	def script_braille_toggleSpeakOnRouting(self, gesture):
+		state = config.conf["braille"]["speakOnRouting"] = not config.conf["braille"]["speakOnRouting"]
+		if state:
+			# Translators: The message announced when toggling on speaking character when routing.
+			state = _("Toggled on speak character when routing cursor in text")
```

Comment:
Wouldn't "Turned" be a less awkward word in English? Not sure how well it translates...

```suggestion
			state = _("Turned on speak character when routing cursor in text")
```

Reply:
I don't think we use the word `turn` that way somewhere else.
```suggestion
			state = _("Enabled speak character when routing cursor in text")
```